can/interfaces/usb2can/usb2canInterface.py

```diff
@@ -9,7 +9,6 @@
 from can import BusABC, Message, CanInitializationError, CanOperationError
 from .usb2canabstractionlayer import Usb2CanAbstractionLayer, CanalMsg, CanalError
 from .usb2canabstractionlayer import (
-    flags_t,
```

Comment:
The only reason I removed this is that with this change it is unused in the file.